cmd/cvp-trigger/main_test.go

```diff
@@ -683,3 +686,33 @@ func TestAppendMultiStageParams(t *testing.T) {
 		})
 	}
 }
+
+func Test_getProwWatcher(t *testing.T) {
```

Comment:
In my opinion, the `getProwWatcher()` logic a) does not need to be in a method, as it's not reused and b) is a thin wrapper over two libraries - `wait` and the ProwJob client. Neither really benefit much from testing at a unit level like this.